test/Readout.spec.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable no-undef */
+describe('The readout', function() {
+    let page;
+
+    before(async function() {
+        page = await browser.newPage();
+        await page.goto('http://localhost:8080');
+    });
+
+    after(async function() {
+        await page.close();
+    });
+
+    let checkForSingularElement = async function(selector) {
+        await page.waitFor(selector);
+        expect(await page.$$(selector)).to.have.lengthOf(1);
+    };
+
+    describe('initially', function() {
+
+        it('should show one textarea', async function() {
+            await checkForSingularElement(SEL_TEXTAREA);
+        });
+
+        it('should display instructions in the textarea', async function() {
+            await page.waitFor(SEL_TEXTAREA);
+            expect(await page.$(SEL_TEXTAREA).placeholder).to.not.eql("");
+        });
+
+        it('should have a submit button', async function() {
+            await checkForSingularElement(SEL_SUBMIT);
+        });
+    });
+
+    describe('after clicking submit', function() {
+
+        it('should make a post request', function() {
+            this.skip('This may be ugly to test without stubbing the backend server.');
+        });
+    });
+
+    describe('after receiving a response', function() {
+        const TEXT = 'Some text';
+
+        before(async function() {
+            // Enter text in the textarea
+            await page.type(SEL_TEXTAREA, TEXT);
+
+            // Click the submit button
+            let button = await page.$(SEL_SUBMIT);
+            await button.click();
+        });
+
+        it('should have a back button', async function() {
+            await checkForSingularElement(SEL_BACK);
+        });
+
+        it('should show the original text', async function() {
+            const element = await page.$(SEL_FEEDBACK);
+            const text = await page.evaluate(element => element.innerText, element);
+
+            expect(text).to.include(TEXT);
+        });
+
+        it('should not show the textarea', async function() {
+            let textArea = await page.$(SEL_TEXTAREA);
```

Comment:
Whenever possible, use `const` instead of `let`. You should not just fix it here, but go back and review all the tests for consistency.